listish.py

```diff
@@ -4,11 +4,23 @@
 """
 
 from collections import Sequence, MutableSequence
+from sys import version_info
+
+
+if version_info < (3, 0):  # pragma: no cover
+    range = xrange  # pylint: disable=redefined-builtin,invalid-name
```

Comment:
Not super happy with the pylint exclusions here but making it happy by using a new name seemed worse.
Thoughts or preferences?